spec/factories/targets.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: targets
+#
+#  id         :bigint           not null, primary key
+#  title      :string           not null
+#  radius     :float            not null
+#  lat        :float            not null
+#  lon        :float            not null
+#  user_id    :bigint           not null
+#  topic_id   :bigint           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_targets_on_topic_id  (topic_id)
+#  index_targets_on_user_id   (user_id)
+#
+FactoryBot.define do
+  factory :target do
+    title { Faker::Name.name }
+    radius { Faker::Number.between(from: 1.0, to: 600_000.0).round(2) }
+    lat { Faker::Address.latitude }
+    lon { Faker::Address.longitude }
+    association :user, factory: :user
+    association :topic, factory: :topic
```

Comment:
```suggestion
    topic
```
I think this works as well

Reply:
doesn't have to be explicit?

Reply:
Not necessarily, since FactoryBot detects the factory by its name. This is useful when the factory name is different from the association name.